static/.htaccess

```diff
@@ -7,3 +7,8 @@ AddType  application/owl+xml     .owl
 
 AddType  text/trig               .trig
 AddType  application/n-quads     .nq
+
+RewriteEngine on
+# Switch http to https
+RewriteCond     %{HTTPS} !=on
+RewriteRule     ^(.*)$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
```

Comment:
nit pick :grimacing: no newline at end of file reported here

Reply:
Fixed!